services/web/client/source/class/osparc/desktop/preferences/pages/ConfirmationsPage.js

```diff
@@ -103,11 +103,41 @@ qx.Class.define("osparc.desktop.preferences.pages.ConfirmationsPage", {
       const cbConfirmDeleteStudy = new qx.ui.form.CheckBox(this.tr("Delete a Study"));
       preferencesSettings.bind("confirmDeleteStudy", cbConfirmDeleteStudy, "value");
       cbConfirmDeleteStudy.bind("value", preferencesSettings, "confirmDeleteStudy");
+      cbConfirmDeleteStudy.addListener("changeValue", e => {
+        if (!e.getData()) {
+          const msg = this.tr("\
+            Are you sure you want to disable this confirmation?\
+            Keep in mind that it is not possible to retrieve deleted studies");
+          const win = new osparc.ui.window.Confirmation(msg);
+          win.center();
+          win.open();
+          win.addListener("close", () => {
+            if (!win.getConfirmed()) {
+              cbConfirmDeleteStudy.setValue(true);
+            }
+          }, this);
+        }
+      }, this);
       box.add(cbConfirmDeleteStudy);
 
       const cbConfirmDeleteNode = new qx.ui.form.CheckBox(this.tr("Delete a Node"));
       preferencesSettings.bind("confirmDeleteNode", cbConfirmDeleteNode, "value");
       cbConfirmDeleteNode.bind("value", preferencesSettings, "confirmDeleteNode");
+      cbConfirmDeleteNode.addListener("changeValue", e => {
+        if (!e.getData()) {
+          const msg = this.tr("\
+            Are you sure you want to disable this confirmation?\
+            Keep in mind that it is not possible to retrieve deleted nodes");
```

Comment:
same here, a short warning would be sufficient but not worth a lot of time to change